pytorch_pfn_extras/onnx/unstrip_tensor.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import json
+import math
+from pathlib import Path
+
+import onnx
+import onnx.numpy_helper
+import numpy
+
+from pytorch_pfn_extras.onnx import strip_large_tensor as strip
+
+
+def _unstrip_tensor(tensor: onnx.TensorProto) -> None:
+    meta_dict = {}
+    meta_dict_idx = 0
+    for i, external_data in enumerate(tensor.external_data):
+        if external_data.key != "location":
+            continue
+        try:
+            external_data_dict = json.loads(external_data.value)
+            if external_data_dict.get("type", "") == "stripped":
+                meta_dict = external_data_dict
+                meta_dict_idx = i
+                break
+        except ValueError:
+            continue
+    if not meta_dict:
+        return None
+    ave = meta_dict.get("average", None)
+    var = meta_dict.get("variance", None)
+    if ave is None or var is None:
+        return None
+
+    np_dtype = onnx.mapping.TENSOR_TYPE_TO_NP_TYPE[tensor.data_type]
+    dummy_array = numpy.random.normal(ave, math.sqrt(var), tensor.dims).astype(np_dtype)
+    dummy_tensor = onnx.numpy_helper.from_array(dummy_array)
+    tensor.data_location = onnx.TensorProto.DEFAULT
+    tensor.raw_data = dummy_tensor.raw_data
+    del tensor.external_data[meta_dict_idx]
+
+
+def _unstrip_graph(graph: onnx.GraphProto) -> None:
+    for init in graph.initializer:
+        if not strip._is_stripped_or_set_external(init):
+            continue
+        _unstrip_tensor(init)
+
+    for node in graph.node:
+        for attr in node.attribute:
+            if attr.HasField("g"):
+                _unstrip_graph(attr.g)
```

Comment:
I've thought of also un-stripping tensor attributes like following:
```suggestion
            if attr.HasField("g"):
                _unstrip_graph(attr.g)
            elif attr.HasField("t"):
                _unstrip_tensor(attr.t)
```
But current strip doesn't do it so never mind
https://github.com/pfnet/pytorch-pfn-extras/blob/785cc23138a0a648a83dc90c4f09302dc9f0938d/pytorch_pfn_extras/onnx/strip_large_tensor.py#L59

Reply:
it's better to support `attr.t`. will try dealing with another PR along with `strip` method.